TicTacToe/TicTacToe.cs

```diff
@@ -32,46 +37,114 @@ public static void GetInput()
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            PlaceMark(row, column);
+            CheckForWin();
+            CheckForTie();
+            playerTurn = (playerTurn == "X") ? "O" : "X";
+
         }
 
         public static void PlaceMark(int row, int column)
         {
-        // your code goes here
+
+            //string[] myRow = board[row];
+            //string myColumn = myRow[column];
+            //myRow[column] = "X";
+            board[row][column] = playerTurn;
+
+
+
+
+
+
+            // your code goes here
         }
 
         public static bool CheckForWin()
-        {
-            // your code goes here
+        { // your code goes here
 
+            if (HorizontalWin() || VerticalWin() || DiagonalWin() == true)
+            {
+                Console.WriteLine("Player " + playerTurn + " Won!");
+                Environment.Exit(0);
+            }
             return false;
+
+
         }
 
         public static bool CheckForTie()
         {
+            if (board[0][0] != " " && board[0][1] != " " && board[0][2] != " " && board[1][0] != " " &&
```

Comment:
Good job! 
To put it another way though, you want to check if there are any remaining unplayed spots. There is a way to do this with loops that might lend itself better to understanding what you are wanting to confirm vs checking every spot explicitly (by setting the next of each array). Just think about it some and if you have time to go back to this later, see if you can rework it.